src/index.ts

```
@@ -428,13 +419,13 @@ export function genJsrFromPackageJson({ pkgJSON }: { pkgJSON: PackageJson }): JS
 	} as const satisfies JSRJson;
 
 	/* check the JSR object */
-	const validation = typia.validateEquals<JSRJson>(jsr);
+	const validation = JSRConfigurationSchema.safeParse(jsr);
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)